pkg/vadmin/start_db_vc.go

```diff
@@ -78,5 +78,10 @@ func (v *VClusterOps) genStartDBOptions(s *startdb.Parms, certs *HTTPSCerts) (vo
 	*opts.UserName = vapi.SuperUser
 	opts.Password = &v.Password
 	*opts.HonorUserInput = true
+
+	// timeout option
+	if v.VDB.Spec.RestartTimeout != 0 {
+		opts.StatePollingTimeout = v.VDB.Spec.RestartTimeout
+	}
```

Comment:
Same here. Can we rewrite this?
```suggestion
	opts.StatePollingTimeout = v.VDB.Spec.RestartTimeout
```